webhooks/models/billing_payloads.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum
+from typing import Literal
+
+from pydantic import BaseModel, Field
+
+
+class LagoTopics(str, Enum):
+    CREDENTIAL = "issue_done"
+    PROOF = "proof_done"
+    CRED_DEF = "cred_def"
+    REV_REG_DEF = "rev_reg_def"
+    REV_REG_ENTRY = "rev_reg_entry"
+    ATTRIB = "attrib"
+    REVOCATION = "revoked"
+
+
+class LagoEvent(BaseModel):
+    transaction_id: str
+    external_customer_id: str
+
+
+class CredentialBillingEvent(LagoEvent):
+    code: Literal[LagoTopics.CREDENTIAL] = Field("issue_done")
```

Comment:
Ah, the codes in the Field() should probably use LagoTopics.CREDENTIAL too. Otherwise `issue_done` is duplicated